src/wallets/TransactionHistory.test.tsx

```diff
@@ -199,10 +200,10 @@ test('Send modal shows up', async () => {
   expect(getAllByText(/Send.*/)).toHaveLength(3)
 })
 
+// TODO Add advanced transaction flow test cases and test confirmation screens
```

Comment:
Please fix the test for the basic flow.
You can do the advanced test case later, but please add a Jira task.